<launch>
	<arg name="veh" doc="Name of vehicle. ex: megaman"/>
	<arg name="local" default="false" doc="true for running everything you possibly can on laptop. false for everything running on vehicle."/>
	<arg name="config" default="baseline" doc="Specify a config."/>
	<arg name = "param_file_name" default = "default" />
	<arg name = "classes" default = "3" />
	<arg name = "motion_mode" default = "0" />
	<arg name = "pkg_name" value = "gd_motion" />
	<arg name = "node_name" default = "gd_text_car_cmd_node" />

	<node ns="$(arg veh)" pkg="gd_motion" type="gd_text_car_cmd_node.py" name="gd_text_car_cmd_node" output="screen" clear_params="true" required="true">
		<rosparam command="load" file="$(find guidedog)/config/$(arg config)/$(arg pkg_name)/$(arg node_name)/$(arg param_file_name).yaml"/>
		<param name="~classes" value="$(arg classes)"/>
		<param name="~motion_mode" value="$(arg motion_mode)"/>
    </node>
</launch>
